add bytesize for nf-core-modules-dev (#4149)
nf-core/website: sites/main-site/src/content/events/2026/bytesize_modules_claude_plugin.md

---

> title: "Bytesize: claude plugins for modules"

Reviewer: ```suggestion
title: "Bytesize: Claude code plugin for nf-core modules"
```

subtitle: nf-core, modules, claude, code
type: talk
startDate: "2026-04-21"
startTime: "13:00+01:00"
endDate: "2026-04-21"
endTime: "13:30+01:00"

> locations:

Reviewer: ```suggestion
startTime: "13:00+02:00"
endDate: "2026-04-21"
endTime: "13:30+02:00"
locations:
```

  - name: Online
    links:
      - https://stockholmuniversity.zoom.us/j/66855293599
---

> This week, Evangelos Karatzas (@vagkaratzas) will showcase his claude plugins for nf-core modules.

Reviewer: ```suggestion
This week, Evangelos Karatzas (@vagkaratzas) will showcase his [Claude Code plugin for nf-core modules](https://github.com/vagkaratzas/nf-core-module-dev).
```

> -> https://github.com/vagkaratzas/nf-core-module-dev

Author: ```suggestion
```
